tests/integration/ha/continuous_writes.py

```diff
@@ -102,7 +103,9 @@ async def _create_fully_replicated_index(self):
             # create index with a replica shard on every node
             client.indices.create(
                 index=ContinuousWrites.INDEX_NAME,
-                body={"settings": {"index": {"auto_expand_replicas": "1-all"}}},
+                body={
+                    "settings": {"index": {"number_of_shards": 2, "auto_expand_replicas": "1-all"}}
```

Comment:
let's create only one shard per index. It will make the test more reliable.
```suggestion
                    "settings": {"index": {"number_of_shards": 1, "auto_expand_replicas": "1-all"}}
```

Reply:
changed mind, please ignore